index.js

```diff
@@ -0,0 +1,67 @@
+import BookClass from './modules/BookClass.js';
+import saveBook from './modules/saveBook.js';
+import listBooks from './modules/listBooks.js';
+
+import { DateTime } from './modules/luxon.min.js';
+
+const listSection = document.querySelector('#list');
+const formSection = document.querySelector('#form-section');
+const contactSection = document.querySelector('#contact-section');
+const listBtn = document.querySelector('#menu-list');
+const addFormBtn = document.querySelector('#menu-addnew');
+const contactBtn = document.querySelector('#menu-contact');
+
+const showList = () => {
+  window.location.reload();
+};
+listBooks();
+
+const date = document.querySelector('.date');
+// const { DateTime } = luxon;
```

Comment:
As a best practice, kindly remove commented pieces of code so that the code base will be more concise.